frappe/core/doctype/submission_queue/submission_queue.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2022, Frappe Technologies and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Submission Queue', {
+    refresh: function(frm) {
+      frm.add_custom_button(__("Unlock"), () => {
+		frm.call("unlock_doc")
+	  })
```

Comment:
```suggestion
      if (frm.doc.status === "Queued") {
          frm.add_custom_button(__("Unlock Reference Document"), () => {
		    frm.call("unlock_doc")
	      });
	  }
```

only show the button when the status is in `Queued` state?

Reply:
👍🏿